pkg/testgridanalysis/testreportconversion/by_platform.go

```diff
@@ -0,0 +1,60 @@
+package testreportconversion
+
+import (
+	sippyprocessingv1 "github.com/openshift/sippy/pkg/apis/sippyprocessing/v1"
+	"github.com/openshift/sippy/pkg/buganalysis"
+	"github.com/openshift/sippy/pkg/testgridanalysis/testgridanalysisapi"
+	"github.com/openshift/sippy/pkg/testgridanalysis/testidentification"
+	"github.com/openshift/sippy/pkg/util/sets"
+)
+
+// convertRawDataToByPlatform takes the raw data and produces a map of platform names to results
+func convertRawDataToByPlatform(
+	rawJobResults map[string]testgridanalysisapi.RawJobResult,
+	bugCache buganalysis.BugCache, // required to associate tests with bug
+	release string, // required to limit bugs to those that apply to the release in question
+	minRuns int, // indicates how many runs are required for a test is included in overall percentages
+	// TODO deads2k wants to eliminate the successThreshold
+	successThreshold float64, // indicates an upper bound on how successful a test can be before it is excluded
+) map[string]sippyprocessingv1.SortedAggregateTestsResult {
+
+	resultsByPlatform := make(map[string]sippyprocessingv1.SortedAggregateTestsResult)
+	for _, platform := range testidentification.AllPlatforms.List() {
+
+		allPlatformTestResults := []sippyprocessingv1.TestResult{}
+		successfulJobRuns := 0
+		failedJobRuns := 0
```

Comment:
these values don't appear to be getting used for anything (they aren't set anywhere or returned out of this function)

Reply:
> these values don't appear to be getting used for anything (they aren't set anywhere or returned out of this function)

they get used one pull later